rtrlib/aspa/aspa.c

```diff
@@ -32,6 +32,11 @@ static enum aspa_status aspa_table_notify_clients(struct aspa_table *aspa_table,
 			size_t size = sizeof(uint32_t) * record->provider_count;
 
 			rec.provider_asns = lrtr_malloc(size);
+
+			if (!rec.provider_asns) {
```

Comment:
I much prefer, the more verbose variant of `if(rec.provider_asns == NULL)`, but this is a personal preference.